src/AutoRest.CSharp.V3/CodeGen/CsProjWriter.cs

```diff
@@ -17,8 +17,7 @@ public bool WriteCsProj(Configuration configuration)
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <OutputPath>bin</OutputPath>
     <PublishDir>$(OutputPath)</PublishDir>
-    <!-- Some methods are marked async and don't have an await in them -->
-    <NoWarn>1998</NoWarn>
+    <NoWarn>SA1649</NoWarn>
```

Comment:
Why do we need this?

Reply:
Because we make files not named the encompassing type. Aka, the `Model.Serialiation.cs` files. Since this is by-design, we can ignore the warning.

Reply:
And, we'll likely make more files like this in the future. For example, XML might be a different file versus JSON serialization.